android/src/com/mapswithme/maps/maplayer/SearchWheel.java

```diff
@@ -160,42 +196,52 @@ public void onResume()
 
   private void toggleSearchLayout()
   {
-    final int animRes;
-    if (mIsExpanded)
-    {
-      animRes = R.animator.show_zoom_out_alpha;
-    }
-    else
+    View searchLayout = getSearchLayout();
+    if (searchLayout != null)
```

Comment:
nit: It looks confusing when we get and check searchLayout var, but use mSearchLayout inside.
I'd make like this:
```
boolean createSearchLayout()
{
  if (mSearchLayout)
    return true;
  ...
}

if (createSearchLayout())
{
// use mSearchLayout
}
```

Reply:
I agree it could be clearer, I will make a new PR for this